src/docs/truffle/reference/truffle-commands.md

```diff
@@ -361,6 +361,8 @@ Options:
 * `<test_file>`: Name of the test file to be run. Can include path information if the file does not exist in the current directory.
 * `--compile-all`: Compile all contracts instead of intelligently choosing which contracts need to be compiled.
 * `--compile-all-debug`: Like `--compile-all`, but compiles contracts in debug mode for extra information.  Has no effect on Solidity <0.6.3.
+* `--compile-none`: Skip all contract compilations, this option is not intended for use with most tests, which will require compilation.
+* `--migrate-none`: Skip all contract migrations, this option could be useful if you will be deploying your own contracts and don't want Truffle to do this automatically for you.
```

Comment:
Another comma splice; I would again suggest splitting this in two.

Reply:
split into two, thx